lua/nvim-tree/view.lua

```diff
@@ -68,9 +68,9 @@ M.View = {
 ---Find a rogue NvimTree buffer that might have been spawned by i.e. a session.
 ---@return integer|nil
 local function find_rogue_buffer()
-  for i = 1, vim.fn.bufnr("$"), 1 do
-    if vim.fn.bufname(i) == "NvimTree" then
-      return i
+  for _, v in ipairs(a.nvim_list_bufs()) do
+    if vim.fn.bufname(v) == "NvimTree" then
```

Comment:
you can also use `a.nvim_buf_get_name(v)` i think :)

Reply:
`nvim_buf_get_name` returns a path, so I thought this was cleaner.